etc/kayobe/stackhpc.yml

```diff
@@ -79,6 +79,10 @@ stackhpc_repo_centos_stream_powertools_version: "{{ stackhpc_repo_distribution }
 stackhpc_repo_centos_stream_storage_ceph_pacific_url: "{{ stackhpc_repo_mirror_url }}/pulp/content/centos/8-stream/storage/x86_64/ceph-pacific/{{ stackhpc_repo_centos_stream_storage_ceph_pacific_version }}"
 stackhpc_repo_centos_stream_storage_ceph_pacific_version: "{{ stackhpc_repo_distribution }}"
 
+# CentOS Stream 8 - Ceph Quincy
+stackhpc_repo_centos_stream_storage_ceph_quincy_url: "{{ stackhpc_repo_mirror_url }}/pulp/content/centos/8-stream/storage/x86_64/ceph-quincy/{{ stackhpc_repo_centos_stream_storage_ceph_quincy_version }}"
+stackhpc_repo_centos_stream_storage_ceph_quincy_version: "{{ stackhpc_repo_distribution }}" ### DON'T FORGET ABOUT ME
```

Comment:
Not seeing where this comes from

Reply:
my `### DON'T FORGET ABOUT ME` comment didn't help :(

Reply:
Should be all there now

Reply:
Relevant conversation mostly moved [here](https://github.com/stackhpc/stackhpc-kayobe-config/pull/424#discussion_r1126175176)